src/etc/rc.monitor

```diff
@@ -37,3 +37,13 @@ fi
     "openvpn reload ${GATEWAY}" \
     "dyndns reload ${GATEWAY}" \
     "rfc2136 reload ${GATEWAY}"
+
+ALARM_ADDR="${2}"
+ALARM_FLAG="${3}"
+ALARM_RTT="${4}"
+ALARM_RTTSD="${5}"
+ALARM_LOSS="${6}"
+
+if [ -z "$#" -eq 6 ]; then
```

Comment:
-z only takes one argument and "sh -n src/etc/rc.monitor" should return a syntax error... best to compare a string here right away: [ "${#}" = "6" ]

Reply:
How about if [ "$#" -eq 6 ]

Reply:
works too, practically the same as the string match minus the certainty of how "$#" is converted into a number... :)